src/bridging/providers/across/util.ts

```diff
@@ -0,0 +1,69 @@
+import { TargetChainId } from '../../../chains'
+import { SuggestedFeesResponse } from './AcrossApi'
+import { AcrossQuoteResult } from './AcrossBridgeProvider'
+import { AcrossChainConfig, ACROSS_TOKEN_MAPPING } from './const/tokens'
+
+export function getChainConfigs(
+  sourceChainId: TargetChainId,
+  targetChainId: TargetChainId
+): { sourceChainConfig: AcrossChainConfig; targetChainConfig: AcrossChainConfig } | undefined {
+  const sourceChainConfig = getChainConfig(sourceChainId)
+  const targetChainConfig = getChainConfig(targetChainId)
+
+  if (!sourceChainConfig || !targetChainConfig) return
+
+  return { sourceChainConfig, targetChainConfig }
+}
+
+function getChainConfig(chainId: number): AcrossChainConfig | undefined {
+  return Object.values(ACROSS_TOKEN_MAPPING).find((config) => config.chainId === chainId)
```

Comment:
Nitpick, result of `Object.values(ACROSS_TOKEN_MAPPING)` can be cached

Reply:
Actually it can be very simple: `return ACROSS_TOKEN_MAPPING[chainId]`

Reply:
I did the mapping precisely for that, I guess I forgot to use it.. 🙈

Reply:
Fixed 613e7982b8b83ebe3b41e992c980757